new-designer-manual.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to the Animint2 Designer Manual web site!
+This manual explains how to design and create interactive data visualizations using the R package animint2. This package is an updated version of [animint](http://members.cbio.mines-paristech.fr/~thocking/animint-book/Ch00-preface.html)
+
+## What's animint2
+
+Chapter 1 | Preface
+          | What's new? Comparison with animint
+          
+Chapter 2 | animint2
+          | Inside animint2. Dependencies
+          | Structure of the pacakge
+          | Possibilities
+            
+Chapter 3 | ggplot2 and animint2
+
+## animint2 and the grammar of graphics
+
+Chapter 4 | What's the grammar of graphics? Overview of data analysis and visualization.
+
+Chapter 5 | animint2, the basics. How to create an interactive graphic from ggplot2 in seconds.
+
+Chapter 6 | Advanced options
```

Comment:
Personally I wouldn't call clickSelects and showSelected "advanced" options -- they really are the basic interactive tools that animint provides